.github/workflows/docker.yml

```diff
@@ -0,0 +1,22 @@
+name: docker
+
+on:
+  push:
+    branches:
+      - "main"
+  workflow_dispatch:
```

Comment:
This allows to trigger the job manually as I don't think it makes sense to trigger it the moment we merge something, as the runtime will still be building. Maybe we can move it after the the job steps added by #22 but it should be enough for doing some basic testing for now I guess